webapp_frontend/src/background/constants.ts

```diff
@@ -12,7 +12,7 @@ const prod: constantsdef = {
 const dev: constantsdef = {
     url: {
         //API_URL: 'https://cors.filefighter.de/http://filefighter.de/api',
-       // API_URL: 'https://cors-anywhere.herokuapp.com/http://filefighter.ddns.net:3001/api',
+        // API_URL: 'https://cors-anywhere.herokuapp.com/http://filefighter.ddns.net:3001/api',
         API_URL: 'http://localhost:8080',
```

Comment:
Intentionally?

Reply:
yes, the new version is not on the server, or is it?

Reply:
not yet. I will merge the pr soon. so the endpoint can be used.

Reply:
Pr is merged, be sure to test it.